### ISSUE TITLE
react/require-render-return: false positive in stateless arrow function

### ISSUE THREAD

**Issue by teddybradford** [yannickcr/eslint-plugin-react#551]
Given the following code, I get the "react/require-render-return" warning. Since the return is implicit with arrow functions, I don't think this warning should be triggering, right?

```jsx
export const Home = () => (
  <div>
    <span>Hello, world!</span>
  </div>
);
```

**Comment by ljharb** [comment]
This is a duplicate of #550 and is fixed in latest master.
